app/src/main/kotlin/com/github/gotify/messages/MessagesActivity.kt

```diff
@@ -171,52 +201,56 @@ internal class MessagesActivity :
 
     private fun refreshAll() {
         CoilInstance.evict(this)
-        startActivity(Intent(this, InitializationActivity::class.java))
-        finish()
+        messagesViewModel.onRefreshAll()
```

Comment:
Revert this back to the cal to the InitializationActivity, as this will also reset the WebSocketConnection.